src/components/00-materials/story.js

```diff
@@ -36,12 +36,13 @@ const mapToIconItem = icon => {
 };
 
 export default {
-  title: 'Others/Materials',
+  title: 'Brand Elements/Icons and Images',
```

Comment:
How about "brand identity" ? :)

Reply:
With `brand identity` also values etc. of a brand  would be included. `Brand Elements` just includes the core elements which would be a bit more limiting. But I don't have a strong opinion about this :)